modules/aws/s3_test.go

```diff
@@ -29,6 +29,24 @@ func TestCreateAndDestroyS3Bucket(t *testing.T) {
 	DeleteS3Bucket(t, region, s3BucketName)
 }
 
+func TestGetTargetLoggingBucketForS3Bucket(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	id := random.UniqueId()
+	logger.Logf(t, "Random values selected. Region = %s, Id = %s\n", region, id)
+
+	s3BucketName := "gruntwork-terratest-" + strings.ToLower(id)
+	s3LogsBucketName := s3BucketName + "-logs"
+
+	CreateS3Bucket(t, region, s3BucketName)
+	CreateS3Bucket(t, region, s3LogsBucketName)
+	PutS3BucketLogging(t, region, s3BucketName, s3LogsBucketName)
```

Comment:
If we end up removing this function, you should implement this inline. Alternatively, you should enhance https://github.com/gruntwork-io/terratest/tree/master/examples/terraform-aws-s3-example with access logging and update https://github.com/gruntwork-io/terratest/blob/master/test/terraform_aws_s3_example_test.go

Reply:
Yeah - if it's fine for the putting bucket logging to live inside the test only, as we don't need it for more than one scenario, I think it's better to go with it being inline! This way, we don't add extra logic into terratest, but I'm still able to test the `getLogging` function, which _was_ my original and main intention.

Reply:
Coming back to this after a day, I actually think it might be better to do this in terraform (the alternative I suggested). That would probably be a more robust test and cleaner.